src/plugins/intel_npu/tools/protopipe/README.md

```diff
@@ -515,67 +515,72 @@ Iteration <number>:
 
 ## How to build
 ### Prerequisites
-1. Clone `npu-plugin` repository
-2. Build OpenCV G-API with OpenVINO/ONNXRT support
-#### Build OpenCV G-API with OpenVINO/ONNXRT support
-1. Clone OpenCV repo:
+1. Build OpenCV G-API with OpenVINO/ONNXRT support
+- Clone and build [OpenVINO](https://github.com/openvinotoolkit/openvino) from sources
+    ```
+    mkdir "build" && cd "build"
+    cmake ../ -DCMAKE_BUILD_TYPE=Release     ^
+              -DENABLE_PLUGINS_XML=ON        ^
+              -DCMAKE_INSTALL_PREFIX=install ^
+              -DENABLE_DEBUG_CAPS=ON         ^
+              -DENABLE_NPU_DEBUG_CAPS=ON ..
+
+    cmake --build . --config Release --target install --parallel
+    ```
+- Init OpenVINO enviroment
+    ```
+     "<OpenVINO-install-dir>/setupvars.sh"
+    ```
+2. Build OpenCV
+- Clone OpenCV repo:
     ```
     git clone https://github.com/opencv/opencv
-    cd opencv && git checkout 78195bc3df
+    cd opencv && git checkout 3919f33e21
     ```
-2. Build OpenCV G-API:
-	```
-    mkdir -p build && cd build
-    cmake ../ -DBUILD_LIST=gapi                             \
-              -DCMAKE_BUILD_TYPE=Release                    \
-              -DWITH_OPENVINO=ON                            \
-              -DOpenVINO_DIR=<path-to-openvino-install-dir> \
-              -DWITH_ONNX=ON                                \
-              -DORT_INSTALL_DIR=<path-to-onnxrt-install-dir>
+- Build OpenCV G-API:
+    ```
+    mkdir -p "build" && cd "build"
```

Comment:
Have you checked that you don't have `-p` created folders?

Reply:
I consistently got `-p` folder...

Reply:
Done. I hope I caught them all.